packages/backend/server/src/base/storage/providers/r2.ts

```diff
@@ -23,5 +33,52 @@ export class R2StorageProvider extends S3StorageProvider {
       bucket
     );
     this.logger = new Logger(`${R2StorageProvider.name}:${bucket}`);
+    this.key = this.encoder.encode(config.signKey);
+  }
+
+  private async signUrl(url: URL): Promise<string> {
+    const timestamp = Math.floor(Date.now() / 1000);
```

Comment:
Is the expiration time configured in the CF console under signDomain?

Reply:
yes, 1 hour by default

Reply:
yes, configured in cf